### DISCUSSION TITLE
AFM NdNiO2 tutorial: Issue with Replicating the spin polarized state

### DISCUSSION THREAD

**Discussion by YUBODRZHANG** [TRIQS/solid_dmft#67] [Q&A]
Hello TRIQS/solid_dmft Community,

I am currently experiencing difficulties in achieving the spin polarized state while attempting to replicate the tutorial on **"AFM state of NdNiO2."** Starting with a non-magnetic DFT simulation, the magnetic moments produced by the DMFT are virtually zero, only showing minor numerical fluctuations. 
![mag](https://github.com/TRIQS/solid_dmft/assets/26913291/48d124c5-6799-46ac-adfd-d7a6534b9b03)

Dr. Prof. Hampel previously mentioned the possibility of initiating spin polarization at the DFT level. This approach was effective for me, but I would prefer to introduce spin solely at the DMFT level, in line with Prof. Hampel's recommendation.

I suspect there may be a control flag to guide the DMFT towards the spin-polarized state, but my attempts to manipulate, for example g0_mix, have been unsuccessful. The log file is attached for your reference. [result.soliddmft.txt](https://github.com/TRIQS/solid_dmft/files/13407724/result.soliddmft.txt)

Thanks,
Yubo Zhang
Minjiang University

**Comment by the-hampel** [comment]
Dear @YUBODRZHANG,
Recently we changed the way the `magmom` flag works in the solid_dmft input file. The current value to initiate the magnetic state might be insufficient. Maybe you can try to increase the value: 
```
magmom = -0.001, 0.001
```
to something like:
```
magmom = -0.4, 0.4
```
I wil update the tutorial shortly, after trying the parameters myself.

Best,
Alex

**Reply by the-hampel** [reply]
Hi @YUBODRZHANG ,

I just updated the tutorial with the new config file and parameters: https://github.com/TRIQS/solid_dmft/commit/24b859ed40cdc4157c8cc653732aeb212ac967ee . The documentation page should reflect this shortly.

- Alex

**Reply by YUBODRZHANG** [reply]
Dear Dr. Prof. Hampel,

I greatly appreciate your help! I've successfully reproduced the tutorial by increasing the number of sweeps before measurement to `length_cycle = 3000`.

Best regards,
Yubo
